### PR TITLE
(DOCSP-34109) Trigger Throughput Wording

### PR DESCRIPTION
**Pull request by nlarew**
### Jira

- (DOCSP-34109) Trigger Throughput Wording

### Staged Changes

- [Triggers](https://preview-mongodbnlarew.gatsbyjs.io/atlas-app-services/throughput/triggers/)
- [Database Triggers](https://preview-mongodbnlarew.gatsbyjs.io/atlas-app-services/throughput/triggers/database-triggers/)

### CODE REVIEW

File: source/triggers.txt [hunk 1]
```diff
@@ -76,26 +76,34 @@ constraints as all Atlas Functions.
...
-- Optimize the Trigger function's run time. For example, you might
-  reduce the number of network calls that you make.
+- Optimize the Trigger function's run time behavior. For example, you
+  might reduce the number of network calls that you make.
 
 - Reduce the size of each event object with the Trigger's projection
-  filter. For the best performance, we recommend limiting the size of
-  each change event to 2KB or less.
+  filter. For the best performance, limit the size of each change event
+  to 2KB or less.
```
Comment (tsedgwick) [inline source/triggers.txt:106]: @nlarew does it make sense to link to the $projection section of db triggers for this? or do you think that's discoverable enough when you are on this page?

File: source/triggers.txt [hunk 2]
```diff
@@ -76,26 +76,34 @@ constraints as all Atlas Functions.
...
-.. note::
+- Unordered triggers can process multiple events concurrently, up to
+  10,000 at once by default. If your Trigger data source is an M10+
+  Atlas cluster, you can configure individual unordered triggers to
+  exceed the 10,000 concurrent event threshold. To learn more, see
+  :ref:`Maximum Throughput Triggers <triggers-maximum-throughput>`.
 
-   If the linked data source is a dedicated server (M10+ Tier), 
-   you can configure individual triggers to process up to 50,000 events concurrently.
+Trigger capacity is not a direct measure of throughput or a guaranteed
```
Comment (tsedgwick) [inline source/triggers.txt:92]: 👏🏻

**Review by tsedgwick — COMMENTED** [review]
Thanks for updating this!

quick drive by - this was flagged to me.

File: source/triggers.txt (hunk 1 above)
Comment (nlarew) [reply source/triggers.txt:106]: Couldn't hurt to add a link!

**Comment by dpish** [comment]
LGTM

**Review by laurazhukas — COMMENTED** [review]
LGTM 🚀

**Review by krollins-mdb — APPROVED** [review]
LGTM. Thanks for doing this, @nlarew!

**Pull request merged**